### ISSUE TITLE
NEST3 build: multiple definitions

### ISSUE THREAD

**Issue by ikitayama** [nest/nest-simulator#1360]
I thought this is fixed both in the master or nest-3 branches, is it not the case?

Below is my attempt to build nest-3 with Arm Clang on arm64 host.

```
[ 99%] Built target nest_lib
[ 99%] Linking CXX executable nest
/opt/arm/gcc-8.2.0_Generic-AArch64_RHEL-7_aarch64-linux/lib/gcc/aarch64-linux-gnu/8.2.0/../../../../aarch64-linux-gnu/bin/ld: ../topology/libtopology.a(topology_names.cpp.o):(.bss+0x4): multiple definition of `nest::names::allow_autapses'; ../nestkernel/libnestkernel.a(nest_names.cpp.o):(.bss+0x50): first defined here
/opt/arm/gcc-8.2.0_Generic-AArch64_RHEL-7_aarch64-linux/lib/gcc/aarch64-linux-gnu/8.2.0/../../../../aarch64-linux-gnu/bin/ld: ../topology/libtopology.a(topology_names.cpp.o):(.bss+0x8): multiple definition of `nest::names::allow_multapses'; ../nestkernel/libnestkernel.a(nest_names.cpp.o):(.bss+0x54): first defined here
/opt/arm/gcc-8.2.0_Generic-AArch64_RHEL-7_aarch64-linux/lib/gcc/aarch64-linux-gnu/8.2.0/../../../../aarch64-linux-gnu/bin/ld: ../topology/libtopology.a(topology_names.cpp.o):(.bss+0x58): multiple definition of `nest::names::max'; ../nestkernel/libnestkernel.a(nest_names.cpp.o):(.bss+0x360): first defined here
/opt/arm/gcc-8.2.0_Generic-AArch64_RHEL-7_aarch64-linux/lib/gcc/aarch64-linux-gnu/8.2.0/../../../../aarch64-linux-gnu/bin/ld: ../topology/libtopology.a(topology_names.cpp.o):(.bss+0x5c): multiple definition of `nest::names::min'; ../nestkernel/libnestkernel.a(nest_names.cpp.o):(.bss+0x384): first defined here
/opt/arm/gcc-8.2.0_Generic-AArch64_RHEL-7_aarch64-linux/lib/gcc/aarch64-linux-gnu/8.2.0/../../../../aarch64-linux-gnu/bin/ld: ../topology/libtopology.a(topology_names.cpp.o):(.bss+0x94): multiple definition of `nest::names::theta'; ../nestkernel/libnestkernel.a(nest_names.cpp.o):(.bss+0x68c): first defined here
/opt/arm/gcc-8.2.0_Generic-AArch64_RHEL-7_aarch64-linux/lib/gcc/aarch64-linux-gnu/8.2.0/../../../../aarch64-linux-gnu/bin/ld: ../topology/libtopology.a(topology_names.cpp.o):(.bss+0x68): multiple definition of `nest::names::offset'; ../nestkernel/libnestkernel.a(nest_names.cpp.o):(.bss+0x3fc): first defined here
clang-7: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [nest/nest] Error 1
make[1]: *** [nest/CMakeFiles/nest.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by ikitayama** [comment]
I don't see the errors above with trunk Clang on the same host.

**Comment by stinebuu** [comment]
Thanks for reporting this @ikitayama! It never showed up for me. We forgot to remove the names when moving some of the functionality to nestkernel. I have fixed it with fcb8a81af0fdba90d138b8a586989eef7be8bce9 in the nest-3 branch, could you try again with the newest version and see if the problem is still there?

**Comment by ikitayama** [comment]
Now it builds fine ☺️
